### DISCUSSION TITLE
Welcome to nuwa-pytorch Discussions!

### DISCUSSION THREAD

**Discussion by lucidrains** [lucidrains/nuwa-pytorch#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by YashVardhan-AI** [comment]
Oh my god you are the best.

**Comment by corranmac** [comment]
Missing you in the discord! Looking forward to keep watching your progress on this amazing project though 😊.

Was wondering how you would feel about having discussions similar to our old ones here with some of the other people?
I totally understand if you'd rather keep discussuion here strictly on topic though. All the best.

**Comment by fractaldna22** [comment]
` tell us about what you do with this community.`
i bring vqgan as close as it can come to nuwa functions without actually training anything lol. am following this intensely

**Comment by lucidrains** [comment]
![video_18700_6a1518590c928b661590](https://user-images.githubusercontent.com/108653/158692279-9865d8ff-34a6-4ca7-808e-48d2f1eae5d5.gif)

small NUWA primed with "3 and 0" - cherrypicked of course :D

follow along in the training here https://tinyurl.com/yc4jy5zt

**Reply by lucidrains** [reply]
![video_76300_102cf575adf2c98d49b0](https://user-images.githubusercontent.com/108653/160253784-671de4d2-884b-4243-a89d-a477fa1237a9.gif)

"2 and 0" -  it is starting to learn zero :)

**Reply by lucidrains** [reply]
also figured out that the NUWA 3d nearby attention wasn't the most efficient, as I could use padding to do a causal - conv-like attention. A bit too late to redo the training for this test run, but have it addressed in the latest version of NUWA https://github.com/lucidrains/nuwa-pytorch/commit/649a91677ce097e81d1f41b98f64bf11aa336bd8

**Reply by lucidrains** [reply]
![6-5](https://user-images.githubusercontent.com/108653/161133513-0d28b98a-ac5f-4e68-8eac-b8924f8027ac.gif)

"6 and 5" - Seeing a 5 for a brief second :)

**Comment by tjthejuggler** [comment]
With what is available here are we able to actually create video from my own text? If so, then after I run the training file, what file do i run with my text in, and where do i put my text? Thanks so much!

**Reply by lucidrains** [reply]
try following the instructions in the second half of this section https://github.com/lucidrains/nuwa-pytorch#trainers
